airflow/www/extensions/init_appbuilder.py

```diff
@@ -530,14 +523,6 @@ def add_view_no_menu(self, baseview, endpoint=None, static_folder=None):
             log.warning(LOGMSG_WAR_FAB_VIEW_EXISTS.format(baseview.__class__.__name__))
         return baseview
 
-    def add_api(self, baseview):
-        """
-            Add a BaseApi class or child to AppBuilder
-        :param baseview: A BaseApi type class
-        :return: The instantiated base view
-        """
-        return self.add_view_no_menu(baseview)
```

Comment:
Thoughts? This should help avoid new FAB endpoints showing up automatically.

Reply:
If it’s not used, we delete it 👍 (Although this doesn’t really prevent it from happening since calling `add_view_no_menu` would still make the endpoint show up?)

Reply:
Yeah, it's certainly not foolproof, but the examples and docs in FAB use this method: https://flask-appbuilder.readthedocs.io/en/latest/rest_api.html